vertx-mongo-client/src/main/java/io/vertx/ext/mongo/package-info.java

```diff
@@ -364,6 +364,294 @@
  * {@link examples.Examples#example16_d1}
  * ----
  *
+ * == Storing/Retrieving files and binary data
+ *
+ * The client can store and retrieve files and binary data using MongoDB GridFS. The
+ * {@link io.vertx.ext.mongo.MongoGridFsClient} can be used to send a whole files or buffers
+ * to GridFS. If the object being stored is large, it can be stored one buffer at a time so as
+ * to avoid Out of Memory exceptions.
+ *
+ * === Get the MongoGridFsClient to interact with GridFS.
+ *
+ * The {@link io.vertx.ext.mongo.MongoGridFsClient} is created by calling
+ * {@link io.vertx.ext.mongo.MongoClient#createGridFsBucketService} and providing a bucket name. In GridFS, the bucket name
+ * ends up being a collection that contains references to all of the objects that are stored. By default the
+ * bucket name is "fs". You can segregate objects into distinct buckets by providing a unique name.
+ *
+ * This has the following fields:
+ *
+ * `bucketName`:: The name of the bucket to create
```

Comment:
I made it single, but I noticed that other modules like jdbc are using double to make it appear as a bold label